lighty-examples/lighty-community-aaa-restconf-app/src/main/assembly/resources/sampleConfigSingleNode.json

```diff
@@ -19,18 +19,7 @@
         },
         "schemaServiceConfig":{
             "topLevelModels":[
```

Comment:
This configuration will fail when is started with example. Please fix missing configuration:
Schema for interface org.opendaylight.yang.gen.v1.urn.opendaylight.params.xml.ns.yang.controller.entity.owners.norev.OdlEntityOwnersService is not available